apis/v1alpha2/gateway_types.go

```diff
@@ -518,12 +518,19 @@ const (
 	// Support: Extended
 	IPAddressType AddressType = "IPAddress"
 
-	// An opaque identifier that represents a specific IP address. The
-	// interpretation of the name is dependent on the controller. For
-	// example, a "NamedAddress" might be a cloud-dependent identifier
-	// for a static or elastic IP.
+	// A hostname represents a DNS based ingress point. This is similar to the
```

Comment:
```suggestion
	// Hostname represents a DNS based ingress point. This is similar to the
```
nit: Don't feel too strongly about this one, I like Go's convention more (although this is not following that strictly either).

Reply:
I think @jpeach may have changed the format of these intentionally to make our reference docs a bit more readable. I could be wrong on the history, but it does read slightly better here: https://gateway-api.sigs.k8s.io/references/spec/#networking.x-k8s.io/v1alpha1.AddressType.